### DISCUSSION TITLE
I am having trouble identifying where to put in my addresses and uuids for the multiple connect for the bluetooth devices

### DISCUSSION THREAD

**Discussion by KaustubhPawar0629** [hbldh/bleak#1449] [Q&A] [answered]
So for my school assignment, I am supposed to connect 2 heart beat sensor devices and read out their heart rate measurement and this works with the singular device connection accurately. However when I started to use the multiple devices example file, I couldn't figure out where I am missing to pass in my arguements for the __name__ init function. I tried to put in the metavar(image 1 and top of image 2) and the name or flags(image 1 and bottom of image 2). Would appreciate the help!
![Screenshot from 2023-11-09 11-24-03](https://github.com/hbldh/bleak/assets/143557438/1170ae05-04fb-45eb-ba35-4611bf7e83c9)
![Screenshot from 2023-11-09 11-24-48](https://github.com/hbldh/bleak/assets/143557438/1645d6fd-4814-4373-a55d-6c2f7a2dfcec)

**Comment by dlech** [comment, marked as answer]
If you want to hard-code them in the program, then you don't need argparse. Just replace `args.device1` with `"A0:9E:1A:49:A8:51"`, etc. and delete all of the parser stuff.
